src/coreModules/documentation/utilities/createModelLink.js

```diff
@@ -0,0 +1,3 @@
+export default function createModelLink({ version = 1, modelName }) {
```

Comment:
as discussed yesterday we generally want alphabetical order and/or optional params last :-)

Reply:
true. we should look for a lint rule for that

Reply:
didn't find any